getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1081,40 +1072,41 @@ the ``is`` keyword can be used:
     if (entity is Enemy):
         entity.apply_damage()
 
-To call a function in a *base class* (i.e. one ``extend``-ed in your current class),
-prepend ``.`` to the function name:
+To call a function in a *parent class* (i.e. one ``extend``-ed in your current
+class), prepend ``.`` to the function name::
 
-::
-
-    .basefunc(args)
+    .base_func(args)
 
 This is especially useful because functions in extending classes replace
-functions with the same name in their base classes. So if you still want
-to call them, you can use ``.`` like the ``super`` keyword in other languages:
-
-::
+functions with the same name in their parent classes. If you still want to
+call them, you can prefix them with ``.`` (like the ``super`` keyword
+in other languages)::
 
     func some_func(x):
-        .some_func(x) # Calls same function on the parent class.
+        .some_func(x) # Calls the same function on the parent class.
+
+.. note:: Default functions like  ``_init``, and most notifications such as
+          ``_enter_tree``, ``_exit_tree``, ``_process``, ``_physics_process``,
+          etc. are called in all parent classes automatically.
+          There is no need to call them explicitly when overloading them.
+
 
 Class Constructor
 ^^^^^^^^^^^^^^^^^
 
-The class constructor, called on class instantiation, is named ``_init``.
-As mentioned earlier, the constructors of parent classes are called automatically when
-inheriting a class. So there is usually no need to call ``._init()`` explicitly.
+The class constructor, called on class instantiation, is named ``_init``. As
+mentioned earlier, the constructors of parent classes are called automatically
+when inheriting a class. So, there is usually no need to call ``._init()``
+explicitly.
 
-Unlike the call of a regular function, like in the above example with ``.some_func``,
-if the constructor from the inherited class takes arguments, they are passed like this:
-
-::
+Unlike the call of a regular function, like in the above example with
+``.some_func``, if the constructor from the inherited class takes arguments,
+they are passed like this::
 
     func _init(args).(parent_args):
        pass
 
-This is better explained through examples. Say we have this scenario:
-
-::
+This is better explained through examples. Say, we have this scenario::
```

Comment:
The comma is incorrect here. Alternatively, you could say "Consider this scenario"